embedded-database-spring-test/src/test/java/io/zonky/test/db/provider/postgres/YandexPostgresDatabaseProviderTest.java

```diff
@@ -77,7 +77,7 @@ public void testGetDatabase() throws Exception {
     @Test
     public void testConfigurationProperties() throws Exception {
         MockEnvironment environment = new MockEnvironment();
-        environment.setProperty("zonky.test.database.postgres.yandex-provider.postgres-version", "9.6.11-1");
+        environment.setProperty("zonky.test.database.postgres.yandex-provider.postgres-version", "10.23-1");
```

Comment:
I guess the point of this test is to have a different PostgreSQL version from the default value to verify that changing the property works.